content/docs/guides/reflex.md

```diff
@@ -0,0 +1,223 @@
+2. Click **New Project**.
+3. Specify your project settings and click **Create Project**.
+
+
+## Set up a Reflex project
+
+To set up a Reflex project, you need to install the Reflex CLI and create a new project.
+
+It's recommended to use a virtual environment to manage your project dependencies. In this example, `venv` is used to create a virtual environment. You can use any other virtual environment manager of your choice like `poetry`, `pipenv`, or `uv`.
+
+To create a virtual environment, run the following command in your project directory:
+
+<CodeTabs labels={["MacOS", "Windows"]}>
+
+    ```bash
+    python3 -m venv .venv
+    source .venv/bin/activate
+    ```
+
+    ```
+    py -3 -m venv .venv
+    .venv\Scripts\activate
+    ```
+    
+</CodeTabs>
+
+
+### Install the Reflex CLI
+
+To install the Reflex CLI, run the following command:
+
+```bash
+pip install reflex
+```
+
+### Create a new Reflex project
+
+First, create a project directory for the Reflex app.
+
+```bash
+mkdir new_project
+cd new_project
+```
+
+To initialize the Reflex app, run the following command:
+
+```bash
+reflex init
+```
+
+When a project is initialized, the Reflex CLI creates a project directory. This directory will contain the following files and directories:
+
+```
+<new_project>
+├── .web
+├── assets
+├── <new_project>
+│   ├── __init__.py
+│   └── <new_project>.py
+└── rxconfig.py
+```
+
+The `rxconfig.py` file contains the project configuration settings. This is where the database connection settings will be defined.
+
+
+### Run the Reflex App
+
+To run the Reflex app, use the following command:
+
+```bash
+reflex run
+```
+
+The Reflex server starts and runs on `http://localhost:3000`.
+
+
+## Configure Reflex connection settings
+
+Now that you have set up a Reflex project, you can configure the connection settings to connect to Neon.
+
+To configure the connection settings:
+
+1. Open the `rxconfig.py` file in the project directory.
+
+2. Adjust the following code in the `rxconfig.py` file to match your Neon connection details:
+
+    ```python
+    # rxconfig.py
+    import reflex as rx
+
+    config = rx.Config(
+        app_name="new_project",
+        # Connect to your own database.
+        db_url="<connection-string-from-neon>",
+    )
+    ```
+
+    Replace `<connection-string-from-neon>` with your Neon connection string. You can find all of the connection details listed above in the **Connection Details** widget on the Neon **Dashboard**. For more information, see [Connect from any application](/docs/connect/connect-from-any-app).
+
+    <Admonition type="note">
+
+    Replace the value for `db_url` with an environment variable or the connection string from Neon. For example, after creating an environment variable named `NEON_DATABASE_URL`, you can use it as follows:
+
+    ```python
+    import os
+
+    DB_URL = os.getenv("NEON_DATABASE_URL")
```

Comment:
Just realized this code is incorrect on reading it through again. It should look like this:

```
database_url = os.getenv("NEON_DATABASE_URL", "sqlite:///reflex.db")

config = rx.Config(
    app_name="new_project",
    db_url=database_url,
    )
```

Reply:
The intent here would be to have Neon as the default and throw if not present. Just in the context of the guide.